dbt/models/iasworld/schema/iasworld.dweldat.yml

```diff
@@ -221,6 +232,14 @@ sources:
               - accepted_values:
                   name: iasworld_dweldat_cur_in_accepted_values
                   values: ['Y', 'D']
+                  additional_select_columns: *select-columns
+                  config:
+                    where: |
+                        CAST(taxyr AS int) BETWEEN {{ var('test_qc_year_start') }} AND {{ var('test_qc_year_end') }}
+                        AND deactivat IS NULL
+                        AND class NOT IN ('201', '213', '218', '219', '220', '221', '224', '225', '236', '240', '241', '290', '294', '297')
```

Comment:
**question (non-blocking):** What's the need for the additional class filter on `cur` in `dweldat`?

Reply:
This is just copied from the `unique-conditions` definition for `dweldat`:

https://github.com/ccao-data/data-architecture/blob/d7c7fc05df3ea1635b5efbbe593288b7f5a090c6/dbt/models/iasworld/schema/iasworld.dweldat.yml#L61-L66

But it seems potentially unnecessary in this case, so I'll plan to remove it along with the `deactivat` filter.

Reply:
Done in 72cb53c188175b3b53e5f12326e902837270fc39.